### ISSUE TITLE
Sort output of xrefs list

### ISSUE THREAD

**Issue by neobrain** [radare/radare2#6201]
The output of commands like `axt` and Vx are very hard to parse (e.g. for functions that have more than 10 callers) because the references seem to be listed in no "obvious" order[1]. It would be helpful if the output of these commands were sorted by the caller address or the caller function name instead.

[1] I suppose the order is just determined by the analysis. Hence, the list appears to be "partially" ordered in many cases.

**Comment by radare** [comment]
use the sort operator

**Comment by neobrain** [comment]
@Maijin Why was this issue closed?

**Comment by Maijin** [comment]
@radare answered, use sort operator ?

**Comment by neobrain** [comment]
@Maijin @radare After friendly explanation by @Maijin on IRC, I still haven't been able to get the output sorted via "axt~$[1]" because that seems to only print a single (sorted) column (rather than the command output sorted by a particular column).

In any case, I still think the output of these commands should be changed because:

* The sort operator cannot be applied manually on Vx, as far as I know
* Requiring people to memorize the sorting operator sequence to make "axt" usable at all for large numbers of references is not ergonomic

**Comment by radare** [comment]
I dont see why column sorting should do what you say. Can you show an asciinema or copypaste/screenshot of that behaviour?

Rcons sorts rows depending on a specific column. It will not mix data between them

**Comment by Maijin** [comment]
ping @neobrain

**Comment by neobrain** [comment]
@Maijin @radare I don't quite get the fuzz; neither axt nor Vx have their output sorted, and unsorted lists are less usable than sorted ones. Not sure how this needs any further illustration?

**Comment by Maijin** [comment]
As pancake said: "Can you show an asciinema or copypaste/screenshot of that behaviour?"

**Comment by Maijin** [comment]
Will reopen when you provide stuff then.
